### ISSUE TITLE
envFile option doesn't work in dap-dlv-go

### ISSUE THREAD

**Issue by Daut** [emacs-lsp/dap-mode#638]
Hi,

I can't load env variables using `envFile` option with `dap-dlv-go`. `dap-go` works fine.

Here is my launch.json file:
```JSON
{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "go",
      "request": "launch",
      "name": "Debug Project - Launch",
      "mode": "auto",
      "program": "${workspaceFolder}/main.go",
      "envFile": "${workspaceFolder}/.env"
    }
  ]
}
```

**Comment by s-kostyaev** [comment]
Looks like delve supports only `env` option and doesn't support `envFile` - https://pkg.go.dev/github.com/go-delve/delve/service/dap#LaunchConfig

**Comment by s-kostyaev** [comment]
Maybe we can reimplement VS Code solution - parse env file and put it into `env` field. But I have no time for next 3 week. Maybe someone else try to implement it.

**Comment by s-kostyaev** [comment]
Workaround for now:
```
{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "go",
      "request": "launch",
      "name": "Debug Project - Launch",
      "mode": "auto",
      "program": "${workspaceFolder}/main.go",
      "env": {
         "TEST_VAR": "foo"
      }
    }
  ]
}
```

**Comment by s-kostyaev** [comment]
@Daut could you test fix from #643 ?

**Comment by Daut** [comment]
@s-kostyaev it works now. Thanks!
